src/main/resources/application.yml

```diff
@@ -238,6 +237,10 @@ payment-provider:
       access-key: f5cd942d-5b23-4467-80ce-95cd67186c6f
       secret-key: ${PAYMENT_SECRET_COOP}
       bic: EKRDEE22
+    tulundusuhistu:
+      access-key: 02f596e6-d6b6-43a8-83ac-53d33cc1f13d
+      secret-key: ${PAYMENT_SECRET_TULUNDUSUHISTU}
```

Comment:
Is the environment variable set up in production?

Reply:
yes